content/docs/apps/leveraging-authentication.md

```diff
@@ -36,14 +36,14 @@ query parameters:
 * `response_type=code`
 * (optional) `state=<ANYTHING>`
 
-You can optionally set a `state` parameter with any value you'd like.
-It will be returned to you in a later step. While optional, it's highly
-recommended that you use it for [security reasons](http://www.twobotechnologies.com/blog/2014/02/importance-of-state-in-oauth2.html).
+You can set a `state` parameter with any value you'd like.
+It will be returned to you in a later step. While optional, we *strongly*
+recommend that you use it with a high-quality random number or a hash generated with a secret key, because it [protects against cross-site request forgery attacks](https://tools.ietf.org/id/draft-bradley-oauth-jwt-encoded-state-07.html).
```

Comment:
My question is why we're linking to a doc about JWTs at all. We're talking about adding a random `state` parameter to requests to prevent CSRF, right?